blackbox/docs/release_notes/2.0.0.txt

```diff
@@ -299,3 +302,19 @@ sets both the minimum and maximum memory to allocate, and should be set to
 whatever your previous ``CRATE_MAX_MEM`` was set to.
 
 .. _enterprise edition: https://crate.io/enterprise-edition/
+
+Cluster name in path data
+-------------------------
+
+The computation of the effective data directory path has changed in a way that
```

Comment:
Not sure how to phrase it. But there shouldn't be any contradiction in the text.

I think it would also be okay to just add in the changes.txt a new entry about the new cluster check (basically a "Added deprecation notices to clusterName usage in `data.path`")

I don't think that people who are already on 2.1 go back to read the 2.0 release notes.